Sources/Fetch/Session+Combine.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Session+Combine.swift
+//  Fetch-iOS
+//
+//  Created by David Hardiman on 21/06/2019.
+//  Copyright © 2019 David Hardiman. All rights reserved.
+//
+
+#if canImport(Combine)
+
+    import Combine
+    import Foundation
+
+    extension Session {
+        @available(iOS 13.0, *)
```

Comment:
Worth matching the `Combine` module interface and doing something like `@available(OSX 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)` ?